kernel/fde/fde.go

```diff
@@ -29,8 +29,11 @@ import (
 	"encoding/json"
```

Comment:
it good to see them in context, but for landing given that this is all a bit still partly untested end-to-end we might want the kernel/fde bits in a separate PR?  cc @mvo5

Reply:
sure we can split this up that's okay, I will open a different PR with just the changes to disks without the new Handler/BackResolver for the kernel/fde package, then this one can be just for the kernel/fde stuff

Reply:
Split out just the disks package changes to https://github.com/snapcore/snapd/pull/11035